src/cmakeProject.ts

```diff
@@ -2472,6 +2473,36 @@ export class CMakeProject implements api.CMakeToolsAPI {
     async selectEnvironments() {
         return null;
     }
+
+    async getExpansionOptions(): Promise<ExpansionOptions> {
+        const workspaceFolder: string = this.workspaceContext.folder.uri.fsPath;
+        return {
+            vars: {
+                buildKit: this.activeKit?.name || "",
+                buildType: await this.currentBuildType() || "",
```

Comment:
Is it only the `additionalKits` setting that uses these expansions?  If so, then we should not support substitution for `buildKit` or `buildType` because those are not stable.